File: api/code-samples/python/klaytnDebug/blockchainInspection/setHead.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

number = "0x100"

sdk = OpenSDK(host)
debug_response = sdk.debug.set_head(number)

print(json.loads(debug_response.response.data))

File: api/code-samples/javascript/klaytnDebug/blockchainInspection/setHead.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));

    const number = "0x100"

    sdk.debug.printBlock(number, {}, (err, data, response) => {
        console.log(data);
    });
}
)()